jobrunner/manage_jobs.py

```diff
@@ -493,11 +493,14 @@ def get_log_dir(job):
 def copy_file(source, dest):
     dest.parent.mkdir(parents=True, exist_ok=True)
     ensure_overwritable(dest)
-    # shutil.copy() should be reasonably efficient in Python 3.8+, but if we
-    # need to stick with 3.7 for some reason we could replace this with a
-    # shellout to `cp`. See:
-    # https://docs.python.org/3/library/shutil.html#shutil-platform-dependent-efficient-copy-operations
-    shutil.copy(source, dest)
+
+    with atomic_writer(dest) as tmp:
+        # shutil.copy() should be reasonably efficient in Python 3.8+, but if we
+        # need to stick with 3.7 for some reason we could replace this with a
+        # shellout to `cp`. See:
+        # https://docs.python.org/3/library/shutil.html#shutil-platform-dependent-efficient-copy-operations
```

Comment:
This comment can go now. We're fully committed to 3.8+ everywhere.

Reply:
It's outa here